schematic/models/metadata.py

```diff
@@ -22,7 +22,7 @@
 # we shouldn't need to expose Synapse functionality explicitly
 from schematic.store.synapse import SynapseStorage
 
-from schematic.utils.df_utils import trim_commas_df
+from schematic.utils.df_utils import trim_commas_df, load_df
```

Comment:
Do we still need trim_commas_df?

Reply:
I think we'll be able to do without it now!